contracts/src/core/AlignedLayerServiceManager.sol

```diff
@@ -326,4 +352,11 @@ contract AlignedLayerServiceManager is
         }
         _;
     }
+
+    modifier validVerifierIdx(uint256 verifierIdx) {
+        if (verifierIdx >= 64) {
+            revert VerifierIdxOutOfBounds();
+        }
+        _;
+    }
```

Comment:
```suggestion
```

Reply:
Fixed in [ee0d97d](https://github.com/yetanotherco/aligned_layer/pull/1090/commits/ee0d97d123d0d7a144a19b35c935b4ba4305ff8a)